src/openrct2/scenario/Scenario.cpp

```diff
@@ -313,7 +316,7 @@ static void scenario_day_update()
 
     // Lower the casualty penalty
     uint16 casualtyPenaltyModifier = (gParkFlags & PARK_FLAGS_NO_MONEY) ? 40 : 7;
-    gParkRatingCasualtyPenalty = max(0, gParkRatingCasualtyPenalty - casualtyPenaltyModifier);
+    gParkRatingCasualtyPenalty = std::max(0, gParkRatingCasualtyPenalty - casualtyPenaltyModifier);
```

Comment:
AppVeyor chokes on this. Perhaps do a Math::Max for now?

Reply:
It's likely because MSVC has some macro named `max` that collides with this. `Math::Max` is not affected because it is capitalised, but there's some define you can set to prevent `max` macro being defined, `NOMINMAX` or something along those lines.

I wonder if and why it worked on @IntelOrca's machine.

Reply:
I did this PR on Linux.